hydra-node/src/Hydra/Chain/CardanoClient.hs

```diff
@@ -36,6 +47,13 @@ instance Exception QueryException where
     QueryEraMismatchException EraMismatch{ledgerEraName, otherEraName} ->
       printf "Connected to cardano-node in unsupported era %s. Please upgrade your hydra-node to era %s." otherEraName ledgerEraName
     QueryProtocolParamsConversionException err -> show err
+    QueryProtocolParamsEraNotSupported unsupportedEraName -> show unsupportedEraName
```

Comment:
This error explanation is not very good as it only shows `ByronEra`. Maybe you missed providing a good error message here.